doc/man3/ASN1_TIME_set.pod

```diff
@@ -42,6 +43,11 @@ format. It will be of the format MMM DD HH:MM:SS YYYY [GMT], for example
 structure has invalid format it prints out "Bad time value" and returns
 an error.
 
+ASN1_TIME_to_tm() converts the time B<s> to the standard B<tm> structure.
```

Comment:
Changed `ASN1_TIME_to_tm()` everywhere to `s` to make it consistent with other ASN1_TIME functions.